variables.tf

```diff
@@ -63,6 +63,14 @@ variable "attributes" {
   default = []
 }
 
+variable "stream" {
+  type = object({
+    enabled   = bool
```

Comment:
Er det noe forskjell på `stream_enabled= null` og `stream_enabled=false`? Hvis ikke synes jeg man kan droppe bruken av object her.

Reply:
Nei, det er ingen praktisk forskjell. Forskjellen er at `null` blir default-verdien til provideren, mens `false` er en eksplisitt deaktiverting. I dette tilfelle blir resultatet det samme. 

Jeg synes også dette objektet ble litt rart. Det jeg forsøkte å illustrere var at `view_type` bare gir mening dersom `enabled`er true. 

Du tenker at det kan være to separate variabler akkurat slik det er i resourcen?